ci_scripts/docker-coredeps/deps/build_rust.sh

```diff
@@ -3,6 +3,9 @@
 set -e -x
 
 # Install Rust
-curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly-2019-09-12 -y
+curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly-2019-11-06 -y
```

Comment:
Could we read the `rust-toolchain` file for the `nightly-2019-11-06` string.  e..g
```
curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain $(cat rust-toolchain) -y
```

Reply:
wouldn't this defeat re-building the docker image? It needs to be rebuild when build_rust.sh changes but with your suggestion it wouldn't change.

Reply:
Hmm, good point.  But no one who bumps the rust-toolchain file will remember to bump this file.  Not sure how to solve that other then perhaps a custom CI action which checks these match - let's consider that out of scope here :)